docs/sdk/src/reference_docs/defensive_programming.rs

```diff
@@ -0,0 +1,589 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! As our runtime should _never_ panic; this includes carefully handling [`Result`]/[`Option`]
```

Comment:
```suggestion
//! As our runtime should _never_ panic, we should carefully handle [`Result`]/[`Option`]
```

I'd change the person on this paragraph. Sounds better that way.